index.md

```diff
@@ -105,7 +105,15 @@ For post-midterm content, Imaad made a summary in [flowchart format](./BU231/flo
 
 #### BU 231 (S24) <small>Business Law</small>
 
-[Notes](./BU231/s24-midterm-notes.pdf){:.a#BU231--Notes--Andy} for all content up to the midterm by Andy.
+A selection of notes for content by Andy:
+
+- [Full Notes](<./BU231/notes-andy/BU 231 Notes.pdf>){:.a#BU231--Notes--Andy}
+- [Midterm Content](<./BU231/notes-andy/BU 231 Notes - Midterm.pdf>){:.a#BU231--Notes--Andy}
+- [Tort Law](<./BU231/notes-andy/BU 231 Notes - Tort.pdf>){:.a#BU231--Notes--Andy} - Intentional and Unintentional Torts, Damages, Liability
+- [Contract Law I](<./BU231/notes-andy/BU 231 Notes - Contract I.pdf>){:.a#BU231--Notes--Andy} - Elements to a Contract, Impeaching Contracts, Requirement of Writing
+- [Contract Law II](<./BU231/notes-andy/BU 231 Notes - Contract II.pdf>){:.a#BU231--Notes--Andy} - Interpretation, Privity, Assignment, Discharge, Remedies
+- [Relationships](<./BU231/notes-andy/BU 231 Notes - Relationships.pdf>){:.a#BU231--Notes--Andy} - Agency, Franchise, Employment, Secured Transactions
+- [Business and Misc.](<./BU231/notes-andy/BU 231 Notes - Business and Misc.pdf>){:.a#BU231--Notes--Andy} - Corporate Structure/Governance, IP, Privacy, International Law
```

Comment:
Instead of having lots of separate files to maintain, can we link directly to pages in the master PDF? In LaTeX you can link to `file.pdf#chapter.1`. It looks like Typst supports this as of last February (typst/typst#2954), not sure what the syntax is.

In the worst case, you can do `file.pdf#page=10`.

Reply:
Ah sorry for the wait

I could not for the life of me figure out the named destinations - i put labels as they said and it didn't seem to work, so I just resorted to page links.